app/server/controller/handler/handlers.go

```diff
@@ -236,3 +237,17 @@ func GetGithubContributorData(w http.ResponseWriter, r *http.Request) {
 	writeHeaders(&w, responseStatusCode)
 	fmt.Fprint(w, string(response))
 }
+
+// GetCommunityData returns all the analytics data related to the community
+func GetCommunityData(w http.ResponseWriter, r *http.Request) {
+	data,err :=community.GetAnalytics()
```

Comment:
```suggestion
	data, err := community.GetAnalytics()
```

Reply:
Yes, corrected it. Go fmt didn't work properly last time.